src/components/SkillPanel.tsx

```diff
@@ -17,11 +17,13 @@ const SkillPanel = ({
   skill,
   context,
   count,
+  categoryLabel,
   onEditClick,
 }: {
   skill: Skill;
   context: string;
   count?: number;
+  categoryLabel?: string;
```

Comment:
I think this parameter should be required, otherwise the url links will contain `undefined`